### DISCUSSION TITLE
USB serial support

### DISCUSSION THREAD

**Discussion by jkmrs** [pascallanger/DIY-Multiprotocol-TX-Module#585] [Q&A]
Hi!
I'm trying to upgrade my USB enabled iRX4 Lite from 1.3.1.1. to the latest FW, 1.3.2.77.
Connecting via Flash Multi, I noticed that my current FW was labelled "USB serial support - TRUE" whereas the on I've downloaded from here https://downloads.multi-module.org/ ,(multi-stm-serial-aetr-v1.3.2.77.bin)  shows USB serial support - FALSE.

The last thing I want is to get stuck with a module I can't upgrade through the built in USB... Where do I find a pre-compiled firmware with USB enabled? Or do I have to go the IDE/compile myself route?

Thanks!

**Comment by benlye** [comment]
USB serial support was dropped to save space.  As long as you update the bootloader on your module you will be able to continue to update it via the USB port.

Read this page for more details:
https://github.com/benlye/flash-multi/blob/master/doc/New_Bootloader.md

**Reply by jkmrs** [reply]
Wow that was fast! :) Thanks! So I'll update the bootloader then flash the new FW.
Again, thanks a lot for all you work on this! Any way to send you a beer or something?? :)
